### PR TITLE
chore(docs): update main README.md and XFS quota docs

### PR DESCRIPTION
**Pull request by niladrih**
Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

Changes:
1. README.md changes
       i. Project status is not GA
      ii. Stability badge not reports GA status
     iii. All docs links now point to openebs.io/docs
     iv. Update to compatibility matrix --> include 3.0.x and exclude 2.10.x
2. XFS Quota docs changes
      i. Add markdown page for prerequisites
     ii. Update sample storageClass as per changes in #107 
    iii. Hide advanced options from plain view
    iv. Add instructions for identifying, and configuring mount options

### CODE REVIEW

File: docs/tutorials/hostpath/xfs_quota/enable-xfs-quota.md [hunk 1]
```diff
@@ -71,42 +37,78 @@ metadata:
...
+  ```
+  
+  `softLimitGrace` and `hardLimitGrace` with PV Storage Request will decide the soft limit and hard limit to be set beyond the storage capacity of the PV.
+  
+  The size of a limit will be as follows:<br>
+  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Size of PV storage request * ( 1 + LimitGrace% )
 
-#### 2. Create a PVC with storage class
+  Setting no value defaults to --> softLimitGrace: "0%" / hardLimitGrace: "0%"<br>
+  This limits capacity to the what was specified in the PV storage request.<br>
```
Comment (Ab-hishek) [inline docs/tutorials/hostpath/xfs_quota/enable-xfs-quota.md]: This sentence seems off to me gramatically.

File: docs/tutorials/hostpath/xfs_quota/prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# Prerequisites for enabling XFS Quota on Hostpath LocalPV
```
Comment (Ab-hishek) [inline docs/tutorials/hostpath/xfs_quota/prerequisites.md:1]: Can we link this prerequisites doc to the enable-xfs-quota.md prerequisite section somehow? Not sure whether that would a great idea. But the details seem to be distributed in various document although they should be but IMO linking them somehow would ease things up.

File: docs/tutorials/hostpath/xfs_quota/use-xfs-fs-with-loop-device.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Create XFS filesystem at the basepath as loop device (if filesystem is not XFS)
```
Comment (Ab-hishek) [inline docs/tutorials/hostpath/xfs_quota/use-xfs-fs-with-loop-device.md:1]: Similarly we can add link to this doc specifying how to bring up a test environment to make use of the quota feature?

File: docs/tutorials/hostpath/xfs_quota/prerequisites.md (hunk 2 above)
Comment (niladrih) [reply docs/tutorials/hostpath/xfs_quota/prerequisites.md:1]: The prerequisites doc's main sub-topics are linked to enable-xfs-quota.md's prerequisites pointers.

Comment (Ab-hishek) [reply docs/tutorials/hostpath/xfs_quota/prerequisites.md:1]: Okay. Yeah! somehow missed it. That seems good.

File: docs/tutorials/hostpath/xfs_quota/use-xfs-fs-with-loop-device.md (hunk 3 above)
Comment (niladrih) [reply docs/tutorials/hostpath/xfs_quota/use-xfs-fs-with-loop-device.md:1]: Added a note section with a link to the loop device setup.

File: docs/tutorials/hostpath/xfs_quota/enable-xfs-quota.md (hunk 1 above)
Comment (niladrih) [reply docs/tutorials/hostpath/xfs_quota/enable-xfs-quota.md]: Modified this sentence. Does the change fix this?

**Review by Ab-hishek — APPROVED** [review]

**Review by kmova — APPROVED** [review]

**Pull request merged**